meshtastic/tests/test_ble_errors_coverage.py

```
@@ -0,0 +1,536 @@
+        """Non-google ImportError should propagate for non-google modules."""
+        # Test the logic path used in the actual import error handling
+        exc = ImportError("Some other error")
+        exc.name = "some.other.module"
+        # This should be re-raised
+        with pytest.raises(ImportError, match="Some other error"):
+            if exc.name not in (
+                "google",
+                "google.protobuf",
+                "google.protobuf.message",
+            ):
+                raise exc
```

Ruff (0.15.7)

[warning] `pytest.raises()` block should contain a single simple statement

(PT012)